### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by SaifAqqad** [erobertson42/bootstrap-modbox#19] [Show and tell]
The wrapper is awesome, it saved me A LOT of time, thank you so much 🤗

**Comment by erobertson42** [comment]
You are quite welcome.  And thank you for saying so, I'm glad you found it useful.

**Comment by femto-code** [comment]
Same here - saved a lot of time of writing own routines. Thanks for publishing this promise-based approach. I agree that almost all other wrapper for modals are outdated and limited.
Thanks again!
